batch/kueue-cohort/team-a-cq.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_batch_kueue_cohort_team_a_cq]
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: team-a-cq
+spec:
+  cohort: all # team-a-cq and team-b-cq share the same cohort
+  namespaceSelector: {}
```

Comment:
this namespaceSelector should match the team-a namespace.

This way, users from team-b can't directly submit jobs here (they can still borrow, through team-b-cq)

Reply:
```yaml
namespaceSelector:
  matchLabels:
    kubernetes.io/metadata.name: team-a
```